framework/test/unit/node/consensus/certificate_generation/commit_pool.spec.ts

```diff
@@ -333,6 +690,39 @@ describe('CommitPool', () => {
 	describe('getAggregageCommit', () => {
 		it.todo('');
 	});
+	describe('_getMaxRemovalHeight', () => {
+		let blockHeader: BlockHeader;
+		const finalizedHeight = 1010;
+
+		beforeEach(() => {
+			chain.finalizedHeight = finalizedHeight;
+
+			blockHeader = createFakeBlockHeader({
+				height: finalizedHeight,
+				timestamp: finalizedHeight * 10,
+				aggregateCommit: {
+					aggregationBits: Buffer.alloc(0),
+					certificateSignature: Buffer.alloc(0),
+					height: finalizedHeight,
+				},
+			});
```

Comment:
same, prefer to only keep values in `beforeEach` which need to be initialized independently for each test